serving/samples/helloworld-go/Dockerfile

```diff
@@ -3,19 +3,20 @@
 FROM golang as builder
```

Comment:
nit: Or we could use `golang:alpine` here :angel:

Reply:
Make sense, done.